src/setuptools_betterproto/_command.py

```diff
@@ -113,3 +116,50 @@ def run(self) -> None:
 
         _logger.info("compiling proto files via: %s", " ".join(protoc_cmd))
         subprocess.run(protoc_cmd, check=True)
+
+
+class AddProtoFiles(BaseProtoCommand):
+    """A command to add the proto files to the source distribution."""
+
+    def run(self) -> None:
+        """Copy the proto files to the source distribution."""
+        proto_files = self.config.expanded_proto_files
+        include_files = self.config.expanded_include_files
+
+        if include_files and not proto_files:
+            _logger.warning(
+                "Some proto files (%s) were found in the `include_paths` (%s), but "
+                "no proto files where found in the `proto_path` (%s). We are not adding "
+                "the found include files to the source distribution automatically!",
```

Comment:
How _would_ we add them? Maybe a hint in the warning?

Reply:
You mean how would you add the proto files in the `include_paths` manually? I think you don't want to do that, you probably just forgot to configure or misconfigured the `proto_path`. Maybe I can add this to the warning.